wagtail/admin/views/pages/listing.py

```diff
@@ -96,7 +96,9 @@ def index(request, parent_page_id=None):
 
     side_panels = PageSidePanels(
         request,
-        parent_page.specific,
+        parent_page.get_latest_revision_as_object(),
```

Comment:
So we don’t need `specific` when fetching a revision?

Reply:
Nope, revision uses a foreign key to the page model and stores the specific page model's content type in itself.